backend/btrixcloud/basecrawls.py

```diff
@@ -93,16 +88,8 @@ def __init__(
         self.background_job_ops = background_job_ops
         self.page_ops = cast(PageOps, None)
 
-        presign_duration_minutes = int(
-            os.environ.get("PRESIGN_DURATION_MINUTES") or PRESIGN_MINUTES_DEFAULT
-        )
-
-        self.presign_duration_seconds = (
-            min(presign_duration_minutes, PRESIGN_MINUTES_MAX) * 60
-        )
-
         # renew when <25% of time remaining
-        self.expire_at_duration_seconds = int(self.presign_duration_seconds * 0.75)
+        self.expire_at_duration_seconds = int(PRESIGN_DURATION_SECONDS * 0.75)
```

Comment:
Do we want to keep this as configurable? Was this just a merge error?

Reply:
It still is configurable! Just getting imported from models now so that we define it once and then import where we need to use it instead of defining the same thing multiple places

Reply:
Ah ok, missed that somehow!